ts/sagas/contentLoaders.ts

```diff
@@ -58,54 +56,54 @@ export function* watchContentServiceLoadSaga(): Iterator<Effect> {
         typeof getServiceMetadata
       > = yield call(getServiceMetadata, serviceId);
 
-      if (response.isRight() && response.value.status === 200) {
-        yield put(
-          contentServiceLoad.success({ serviceId, data: response.value.value })
-        );
-      } else {
-        const errorDescription = response.fold(
+      if (response.isLeft()) {
+        const error = response.fold(
           readableReport,
           ({ status }) => `response status ${status}`
         );
-        yield put(
-          contentServiceLoad.failure({
-            service_id: serviceId,
-            error: Error(`${errorDescription} - serviceId ${serviceId}`)
-          })
-        );
+        throw Error(error);
       }
 
-      const isFirstServiceLoadingCompleted = yield select(
+      // If the service is loaded for the first time (at first startup or when the
+      // cache is cleaned), the app shows the service list item without badge
+      const isFirstVisibleServiceLoadCompleted = yield select(
         isFirstVisibleServiceLoadCompletedSelector
       );
-
-      // If the service content is loaded for the first time, the app shows the service list item without badge
-      if (!isFirstServiceLoadingCompleted) {
+      if (!isFirstVisibleServiceLoadCompleted) {
```

Comment:
The default behavior is to mark service as read when the user enters the service detail and a related badge is displayed to highlight new loaded services. 

At startup we have the exception: if the user is loading the services massively, anyone of the loaded services should be displayed with the badge (otherwise all services will have a badge)

Removing the condition all the services will be marked as read when loaded and the badge will never be displayed.